include/bbp/sonata/config.h

```diff
@@ -618,6 +618,17 @@ class SONATA_API SimulationConfig
      */
     const nonstd::optional<std::string>& getNodeSet() const noexcept;
 
+    /**
+     * Returns the metadata section
+     */
+    const std::unordered_map<std::string, std::string>& getMetaData() const noexcept;
```

Comment:
since `variantValueType` exists, maybe it could be used here too?

Reply:
Unlike `beta_features` where various types of variables will be used for simulation, I intend that `metadata` holds only string variables as they are only for notes.  I've also addressed that in the sonata [spec](https://sonata-extension.readthedocs.io/en/latest/sonata_simulation.html#metadata).  
But I can change if it is better to allow different types of notes.

Reply:
Ok, that works too - I'd prefer to keep the API relatively stable, so if you think it's going to remain that way, I'm fine with that.

The current parser will fail if a non-string is entered, so people should be aware of the problem.